stdlib/sys/process.gr

```diff
@@ -152,6 +152,11 @@ provide let env = () => {
   let envc = WasmI32.load(envcPtr, 0n)
   let envvBufSize = WasmI32.load(envvBufSizePtr, 0n)
 
+  if (envc == 0n) {
+    Memory.free(envcPtr)
+    return Ok(WasmI32.toGrain(allocateArray(0n)): Array<String>)
```

Comment:
Just to double check this is just a fast path for if there are no args right?

and also is there a reason you chose to allocate an array using `allocateArray` over `[>]`?

```suggestion
    return Ok([>]: Array<String>)
```

Reply:
> this is just a fast path for if there are no args right?

Yes, it also makes sure that the function doesn't break since a `Wasi.environ_get` is run in subsequent code, which wouldn't work with 0 arguments.

> and also is there a reason you chose to allocate an array using allocateArray over [>]?

Not particularly, I can change it